.github/workflows/reusable-macos.yml

```diff
@@ -1,3 +1,5 @@
+name: macOS
```

Comment:
Hmmm, To me, this name is a bit overkill since the filename already shows which platform it’s for.